recipes/etc-local.rb

```diff
@@ -30,6 +30,11 @@
       config_path = site['docroot']
     end
 
+    directory "#{config_path}/app/etc" do
+      recursive true
+      action :create
+    end
+
```

Comment:
`If this value is not specified, existing owners remain unchanged and new owner assignments use the current user (when necessary).` from https://docs.chef.io/resource_directory.html

There is also the resource merging that occurs if the directory resource is defined twice, where the correct owner and group of the directory would be merged in.

I can make it a `not_if` though :)